### ISSUE TITLE
Have a border highlighting warp if it is an active window

### ISSUE THREAD

**Issue by seveibar** [warpdotdev/Warp#4324]
![image](https://github.com/warpdotdev/Warp/assets/1910070/f832923a-0ef5-4656-9497-9fb081918b6b)

![image](https://github.com/warpdotdev/Warp/assets/1910070/a043a6f8-1b17-4fcd-9ad8-924ed9d2f75a)

### Describe the solution you'd like?

I use the i3 window manager, each window has a blue border ("window decoration") surrounding it when it's highlighted. For some reason, this doesn't appear around the warp terminal, so it's difficult for me to determine if it's active, especially if I have multiple warp terminals open. Using keyboard shortcuts to switch between them has no indication which one is selected or active because there isn't a colored window border like other windows.

Many programs provide a way to use the "system" window decoration border, I think having this as an option is a good solution, but also if the warp terminal just had any indicator it was active/keyboard input will go to it would be perfect.

### How important is this feature to you?

2

### Warp Internal (ignore) - linear-label:39cc6478-1249-4ee7-950b-c428edfeecd1

None

**Comment by cayhorstmann** [comment]
I have the same problem with Openbox, and would like an option to show the window manager decorations. Like the "Use system title bar and borders" setting in Chromium. Not a good experience to have a window with no borders, no title bar, and black rounded notches at the corners.

**Comment by yulrizka** [comment]
Yes, when working with multiple window terminal, it's really hard to know which one is active
